nixos/modules/services/networking/consul.nix

```diff
@@ -232,26 +237,24 @@ in
       };
     }
 
-    (mkIf (cfg.alerts.enable) {
-      systemd.services.consul-alerts = {
-        wantedBy = [ "multi-user.target" ];
-        after = [ "consul.service" ];
-
-        path = [ cfg.package ];
-
-        serviceConfig = {
-          ExecStart = ''
-            ${cfg.alerts.package}/bin/consul-alerts start \
-              --alert-addr=${cfg.alerts.listenAddr} \
-              --consul-addr=${cfg.alerts.consulAddr} \
-              ${optionalString cfg.alerts.watchChecks "--watch-checks"} \
-              ${optionalString cfg.alerts.watchEvents "--watch-events"}
-          '';
-          User = if cfg.dropPrivileges then "consul" else null;
-          Restart = "on-failure";
+      (mkIf (cfg.alerts.enable) {
+        systemd.services.consul-alerts = {
+          wantedBy = [ "multi-user.target" ];
+          after = [ "consul.service" ];
+
+          path = [ cfg.package ];
+
+          serviceConfig = {
+            ExecStart = ''
+              ${cfg.alerts.package}/bin/consul-alerts start \
+                --alert-addr=${cfg.alerts.listenAddr} \
+                --consul-addr=${cfg.alerts.consulAddr} \
+                ${optionalString cfg.alerts.watchChecks "--watch-checks"} \
+                ${optionalString cfg.alerts.watchEvents "--watch-events"}
+            '';
+            User = if cfg.dropPrivileges then "consul" else null;
+            Restart = "on-failure";
+          };
         };
-      };
-    })
-
-  ]);
```

Comment:
Please keep closing parentheses matching with their levels of introduction, and the `(mkIf (...` block on the same indentation level as the preceding list element. Otherwise it's difficult to see which elements are in the list.

I suspect an auto-formatter made a wrong decision here.

Reply:
I see.